apps/indexer-coordinator/src/monitor/monitor.service.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020-2022 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { Injectable } from '@nestjs/common';
+import { Cron, CronExpression } from '@nestjs/schedule';
+import axios from 'axios';
+import { DockerService } from '../core/docker.service';
+import { ProjectService } from '../project/project.service';
+import { nodeContainer } from '../utils/docker';
+import { getLogger } from '../utils/logger';
+
+@Injectable()
+export class MonitorService {
+  constructor(private docker: DockerService, private projectService: ProjectService) {
+    this.checkNodeHealth();
+  }
+
+  private readonly nodeUnhealthTimes = 3;
+  private nodeUnhealthTimesMap = new Map<string, number>();
+
+  @Cron(CronExpression.EVERY_5_MINUTES)
+  async checkNodeHealth() {
+    getLogger('monitor').info(`check node health started`);
```

Comment:
[suggestion] create a share monitor logger, for example:
```
private logger = getLogger('monitor');
```

Reply:
refactored as required